scripts/README.md

```diff
@@ -0,0 +1,23 @@
+# Scripts
+
+This directory contains examples, helper scripts and the console for PSIJ.
+
+## psij-console
+
+The console takes an exported JobSpec document and either validates or executes it.
+
+
+```
+usage: psij-consol [-h] [-v] [--debug] {validate,run} ...
```

Comment:
I'm not a huge fan of this name but I can't think of anything better, and we can always change it later I guess.